app/presenters/templates/components/form.latte

```diff
@@ -4,9 +4,17 @@
 </ul>
 
 <table>
-<tr n:foreach="$form->controls as $input" n:class="$input->required ? required">
-	<th>{label $input /}</th>
-	<td>{input $input} <span class=error n:ifcontent>{$input->error}</span></td>
-</tr>
+{foreach $form->controls as $input}
+    <tr n:if="!$input instanceof Nette\Forms\Controls\HiddenField" n:class="$input->required ? required">
+        <th>{label $input /}</th>
+        <td>{input $input} <span class=error n:ifcontent>{$input->error}</span></td>
+    </tr>
+{/foreach}
 </table>
+
+{foreach $form->controls as $input}
```

Comment:
It is not needed because not rendered fields are rendered by [`{/form}`](https://github.com/nette/forms/blob/9d15bbdcf52a98fa785476da81bf0824a340e6b4/src/Bridges/FormsLatte/Runtime.php#L59).